aws-opentelemetry-distro/pyproject.toml

```diff
@@ -26,6 +26,7 @@ classifiers = [
 dependencies = [
   "opentelemetry-api == 1.27.0",
   "opentelemetry-sdk == 1.27.0",
+  "opentelemetry-processor-baggage == 0.48b0",
```

Comment:
nit: move this down with other "0.48b0" version dependencies. makes upgrading dependencies easier.